parser/prism/Translator.cc

```diff
@@ -2641,14 +2641,21 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node, bool preserveCon
             auto keywordRestParamNode = down_cast<pm_keyword_rest_parameter_node>(node);
 
             core::NameRef sorbetName;
+            core::LocOffsets kwrestLoc;
             if (auto prismName = keywordRestParamNode->name; prismName != PM_CONSTANT_ID_UNSET) {
                 // A named keyword rest parameter, like `def foo(**kwargs)`
                 sorbetName = translateConstantName(prismName);
+
+                // The location doesn't include the `**`, only the splatted expression like `kwargs` in `**kwargs`
+                constexpr uint8_t length = std::size("**"sv);
```

Comment:
```suggestion
                constexpr uint8_t length = "**"sv.size();
```

consistency with all other types of this pattern in the codebase.

Reply:
(In general, I prefer method calls over argument-dependent lookup function calls, because the errors and IDE features are better.)

Reply:
Done in 0572bdca6491e4bbf77f94e5b462ae52594e892f, will merge up-stack in #9565